inventory/api/compute/v1/compute.proto

```diff
@@ -646,6 +646,18 @@ message InstanceResource {
   provider.v1.ProviderResource provider = 40 [(ent.edge) = {unique: true}]; // Provider this Instance is provisioned through
   localaccount.v1.LocalAccountResource localaccount = 41 [(ent.edge) = {unique: true}]; // Local Account associated with this Instance
 
+  string existing_cves = 50 [
+    (ent.field) = {
+      optional: true
+      immutable: false
+    },
+    (buf.validate.field).string = {
+      max_bytes: 20000
+      pattern: "^$|^[a-zA-Z-_0-9./:;=@?!#,<>*+~()\" \\n{}\\[\\]]+$"
```

Comment:
I'm not sure about the double escapes, can you validate if they are actually needed?
```suggestion
      pattern: "^$|^[a-zA-Z-_0-9./:;=@?!#,<>*+~()\" \\n{}[]]+$"
```